webapp/channels/src/components/advanced_text_editor/send_button/send_button.tsx

```diff
@@ -43,32 +39,74 @@ const SendButtonContainer = styled.button`
 const SendButton = ({disabled, handleSubmit}: SendButtonProps) => {
     const {formatMessage} = useIntl();
 
-    const sendMessage = (e: React.FormEvent) => {
+    const sendMessage = useCallback((e: React.FormEvent, schedulingInfo?: SchedulingInfo) => {
         e.stopPropagation();
         e.preventDefault();
-        handleSubmit(e);
-    };
+        handleSubmit(e, schedulingInfo);
+    }, [handleSubmit]);
+
+    const sendOnCtrlEnter = useSelector(isSendOnCtrlEnter);
+
+    const sendNowKeyboardShortcutDescriptor = useMemo<ShortcutDefinition>(() => {
+        const shortcutDefinition: ShortcutDefinition = {
+            default: [
+                defineMessage({
+                    id: 'shortcuts.generic.enter',
+                    defaultMessage: 'Enter',
+                }),
+            ],
+            mac: [
+                defineMessage({
+                    id: 'shortcuts.generic.enter',
+                    defaultMessage: 'Enter',
+                }),
+            ],
+        };
+
+        if (sendOnCtrlEnter) {
+            shortcutDefinition.default.unshift(ShortcutKeys.ctrl);
+            shortcutDefinition.mac?.unshift(ShortcutKeys.cmd);
+        }
+
+        return shortcutDefinition;
+    }, [sendOnCtrlEnter]);
 
     return (
-        <SendButtonContainer
-            data-testid='SendMessageButton'
-            tabIndex={0}
-            aria-label={formatMessage({
-                id: 'create_post.send_message',
-                defaultMessage: 'Send a message',
-            })}
-            disabled={disabled}
-            onClick={sendMessage}
-        >
-            <SendIcon
-                size={18}
-                color='currentColor'
-                aria-label={formatMessage({
-                    id: 'create_post.icon',
-                    defaultMessage: 'Create a post',
-                })}
+        <div className={classNames('splitSendButton', {disabled})}>
+            <WithTooltip
+                placement='top'
+                id='send_post_now_tooltip'
+                title={formatMessage({id: 'create_post_button.option.send_now', defaultMessage: 'Send Now'})}
+                shortcut={sendNowKeyboardShortcutDescriptor}
+                disabled={disabled}
+            >
+                <SendButtonContainer
+                    className={classNames('SendMessageButton', {disabled})}
+                    data-testid='SendMessageButton'
+                    tabIndex={0}
+                    aria-label={formatMessage({
```

Comment:
This should probably be `Send Now` as well

Reply:
`Send a message` is more descriptive of the button's functionality than `Send Now`. In case of screen reader, send a message is a better description.

Reply:
Oh, I added the missing aria label on schedule button.

Reply:
I can sort of understand that, but I would think that that should match the tooltip though since the tooltip has the same general purpose as the label. I'm also thinking it's like how the `title` attribute adds a system tooltip and is also read by a screen reader

Reply:
I see, that makes sense. Done.